<launch>

	<node pkg="airsim_ros_pkgs" name="package_delivery_node" type="package_delivery_node" output="screen" />

	<node pkg="airsim_ros_pkgs" name="future_collision" type="future_collision"/>

	<node pkg="airsim_ros_pkgs" name="panic_pcl" type="panic_pcl"/>

	<node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
		<param name="resolution" value="0.8" />
	
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="frame_id" type="string" value="world_enu" />
		
		<!-- maximum range to integrate (speedup!) -->
		<param name="sensor_model/max_range" value="60" />
		
		<!-- data source to integrate (PointCloud2) -->
		<remap from="cloud_in" to="/airsim_node/drone_1/front_left_custom/DepthPlanner/registered/points" />

	</node>
</launch>